libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInfo.kt

```diff
@@ -58,10 +58,15 @@ class LnurlPayInfoJob(
             // Calculate the maximum receivable amount within the fee limit (in millisatoshis)
             val feeLimitMsat: ULong = config.channelFeeLimitMsat
             val nodeInfo = breezSDK.nodeInfo()
-            val maxReceivableMsatFeeLimit = minOf(
-                nodeInfo.maxReceivableMsat,
-                feeLimitMsat / (ofp.proportional / 1000000u),
-            )
+            val proportionalPercent = ((ofp.proportional.toDouble() / 1000000.0))
+            val maxReceivableMsatFeeLimit = if (proportionalPercent != 0.0 && feeLimitMsat != 0UL) {
```

Comment:
I think it's enough to check only proportionalPercent,. If feeLimitMsat is 0 then the maxReceivableMsatFeeLimit is 0 and the maxSendable will be equal to inboundLiquidityMsat. I think this is correct
```suggestion
            val maxReceivableMsatFeeLimit = if (proportionalPercent != 0.0) {
```

Reply:
That's true but my reasoning was to treat this feature as disabled when feeLimitMsat is set to 0 where in this case `maxReceivableMsatFeeLimit` should resolve to `maxReceivableMsat`.